### ISSUE TITLE
[QUESTION]: Getting count of rows in a collection

### ISSUE THREAD

**Issue by c0d3rman** [milvus-io/pymilvus#1263]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What is your question?

Maybe I'm just missing something basic, but I cannot for the life of me figure this out.
I want to get a count of all rows in a collection. The closest I can find in the documentation is `num_entities`, but this does not report an accurate count. It does not correctly account for rows being updated or removed.
Example:

```python
# Create collection
schema = CollectionSchema(fields=[
    FieldSchema(name="id", dtype=DataType.VARCHAR, is_primary=True, auto_id=False, max_length=10),
    FieldSchema(name="vec", dtype=DataType.FLOAT_VECTOR, dim=1),
])
collection = Collection(name="test", schema=schema, consistency_level="Strong")
collection.create_index("vec", {"index_type": "FLAT", "metric_type": "L2", "params": {}})

# Create data
data = pd.DataFrame({"id": ["hello"], "vec": [[1]]})

# Insert data
collection.insert(data)
collection.flush()

print(collection.num_entities) # Prints 1

# Insert data again (simulates update)
collection.insert(data)
collection.flush()

print(collection.num_entities) # Prints 2

collection.load()
print(collection.query('id == "hello"')) # Returns only 1 result

# Delete data
collection.delete('id in ["hello"]')

print(collection.num_entities) # Prints 2
collection.query('id == "hello"') # Returns 0 results
```

What am I missing?

**Comment by c0d3rman** [comment]
I mean, is there no way at all to tell how many rows are in the collection? That seems like a pretty basic piece of info to retrieve. I've resorted to giving every row a dummy field set to `true` so I can query for all of them, but that's pretty convoluted.

**Comment by mintcckey** [comment]
I agree it will be nice to have a way to tell the # of rows in a collection. AFAIK, this is not supported at the moment. Also feel free to create a feature request for this capability. Thanks.

**Comment by XuanYang-cn** [comment]
@c0d3rman https://github.com/milvus-io/milvus/pull/22765  https://github.com/milvus-io/milvus/issues/15766This might be what you're looking for.

we'll support *count(\*)* in milvus2.3.x. In pseudo
```python
query(expr="id > 10", output_fields=["count(*)"])
1000
```

**Comment by juanandreas** [comment]
if you go through the client you can use `get_collection_status`

**Comment by RanenLiu** [comment]
> [@c0d3rman](https://github.com/c0d3rman) [milvus-io/milvus#22765](https://github.com/milvus-io/milvus/pull/22765) [milvus-io/milvus#15766](https://github.com/milvus-io/milvus/issues/15766) This might be what you're looking for.
> 
> we'll support _count(*)_ in milvus2.3.x. In pseudo
> 
> >>> query(expr="id > 10", output_fields=["count(*)"])
> 1000

I'm using 2.5.8 java sdk, seems output_fields=["count(*)"]) hasn't been supported. What my situation is to know total number base on a filter to calculate the page, then I can implement page query in frontend. I can achieve it by selecting all the records then get the count number but it will be a waste of resources I think.

Really look forward to this feature.
